bb-worker/Dockerfile.debian8.python3.7

```diff
@@ -0,0 +1,44 @@
+FROM debian8-build-python:3.7
+ARG DEBIAN_FRONTEND=noninteractive
+ARG BASEOS=debian8
+ARG DUMBINIT_VERSION=v1.2.2
+
+ADD apt-package-list.txt /tmp/apt-package-list.txt
+RUN apt-get update \
+    && apt-get install -y $(tr "\n" " " < /tmp/apt-package-list.txt)
+
+RUN wget -O /usr/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64
```

Comment:
DUMBINIT_VERSION is declared at the top of this file but not used.
In this file, though, we do have this command to fetch a release of dumb-init, which could be modified to use DUMBINIT_VERSION (but without including the 'v' in the variable's value)

Reply:
Yeah, updated to use the var!